source/_patterns/molecules/interactive/accordion/accordion.twig

```diff
@@ -0,0 +1,24 @@
+{% set tag = tag|default('h2') %}
+{% set classes = [
+  'usa-accordion',
+  'usa-accordion--bordered'
+  ]|merge(classes|default([]))
+%}
+
+<div{{ attributes.addClass(classes) }}>
+  {% for item in accordion %}
+    {% set expanded = loop.first ? 'true' : 'false' %}
+    {% set id = 'accordion-item-' ~ loop.index ~ '-' ~  random() %}
```

Comment:
We do need to get sort of specific here, but this should be good.  Wouldn't want to have the component consumer have to pass in a unique id just to make this work.